lib/warder/coffee_lint_runner.rb

```diff
@@ -0,0 +1,37 @@
+module Warder
+  # responsible for run coffeescript style guide validation
+  class CoffeeLintRunner < Runner
+    require 'warder/coffee_lint_runner/lint_runner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.